HW_10/task 10.2.1.js

```diff
@@ -0,0 +1,14 @@
+// Task 2
+// 1. Создайте функцию delay, принимающую на вход коллбэк функцию и количество милисекунд.
+//     Функция должна исполнить колбэк строго через переданное количество миллисекунд
+//     Пример: delay(() => console.log('hello'), 2000) // Через 2 секунды в консоли появится слово hello
+
+function callback() {
+    return 'hello';
+}
+
+function delay(callback, time) {
+    return setTimeout(() => console.log(callback), time);
+}
+
+delay(callback(), 2000);
```

Comment:
```suggestion
function delay(callback, time) {
   setTimeout(callback, time);
}

delay(callback, 2000);
```

Reply:
Thanks for suggestion. In this case I'll also replace `return 'hello'` in line 7 to `console.log('hello')` in order to show result in console